doc/revise/revise_dynamics.md

```diff
@@ -0,0 +1,101 @@
+# Revision of Project 3: Dynamics of AS Patterns
+
+## Statement of Problem
+The objective of Dynamics Analysis is to classify the active states and inactive states.
+
```

Comment:
In addition to classifying the AS and the IS, you may also want to mention that one of the key objectives is to understand the dynamics/transitions between AS and IS.